### PR TITLE
Create UX Newsletters.md

### PR DESCRIPTION
**Pull request by milapbhojak**

### CODE REVIEW

File: UX-Design/UX Newsletters.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+##Newsletters
+
+1. [Mailchimp UX Newsletter](http://www.theuxnewsletter.com/)
```
Comment (PareshMayani) [inline UX-Design/UX Newsletters.md]: Please also include details/brief about each links, for example https://github.com/PareshMayani/Learning-Resources/blob/master/android/Useful_links.md

**Comment by rathodchintan** [comment]
Nice list.. :)

**Comment by PareshMayani** [comment]
@milaptank are you going to include brief about each links?

**Comment by milapbhojak** [comment]
@PareshMayani Yes, I will add brief as well as other UX resource by tonight.

**Comment by PareshMayani** [comment]
@milapbhojak sure, thanks!

**Comment by PareshMayani** [comment]
@milapbhojak about UX blogs and magazines, please brief about each links, like who is the author, what kind of contents particular blog serves, daily/weekly updates, etc.

**Comment by PareshMayani** [comment]
Thank you @milapbhojak for the contribution. We can not let it go but please put some details along with links!

**Pull request merged**